src/convenience/keyboard.ts

```diff
@@ -415,6 +417,27 @@ export class InlineKeyboard {
     switchInlineCurrent(text: string, query = "") {
         return this.add({ text, switch_inline_query_current_chat: query });
     }
+    /**
+     * Adds a new inline query button. Telegram clients will let the user pick a
+     * chat when this button is pressed. This will start an inline query. The
+     * selected chat will be prefilled with the name of your bot. You may
+     * provide a text that is specified along with it.
+     *
+     * Your bot will in turn receive updates for inline queries. You can listen
+     * to inline query updates like this:
+     * ```ts
+     * bot.on('inline_query', ctx => { ... })
+     * ```
+     *
+     * @param text The text to display
+     * @param query The query object describing which chats can be picked
+     */
+    switchInlineChosenChat(
```

Comment:
I'm not at all bothered by this but just noting..

I guess the naming convention `switch_inline_query` -> `switchInline` and `switch_inline_query_current_chat` -> `switchInlineCurrent` tends to omit `query` and `chat`. By that pattern, I suppose the conforming name would be more like `switchInlineChosen` instead. /shrug

Reply:
Right, will rename. Thanks!